python/py-selenium/Portfile

```diff
@@ -4,31 +4,53 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-selenium
-version             3.141.0
+version             4.1.0
 license             Apache-2
 maintainers         {dstrubbe @dstrubbe} openmaintainer
 platforms           darwin
 
 description         Python language bindings for Selenium WebDriver
-long_description \
-    The selenium package is used to automate web browser interaction \
-    from Python. Several browsers/drivers are supported (Firefox, \
-    Chrome, Internet Explorer), as well as the Remote protocol.
+long_description    The selenium package is used to automate web \
+                    browser interaction from Python. Several \
+                    browsers/drivers are supported (Firefox, Chrome, \
+                    Internet Explorer), as well as the Remote protocol.
 
 homepage            https://www.seleniumhq.org/
-master_sites        pypi:s/${python.rootname}/
-distname            ${python.rootname}-${version}
 
-python.versions     27 37 38 39 310
-
-checksums           rmd160  6175bf1db6e8de270d0c34b9c58649ab1b118bc7 \
-                    sha256  deaf32b60ad91a4611b98d8002757f29e6f2c2d5fcaf202e1c9ad06d6772300d \
-                    size    854669
+python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                    port:py${python.version}-setuptools
+                    port:py${python.version}-pip \
+                    port:py${python.version}-wheel
+
     depends_run-append \
                     port:py${python.version}-urllib3
+
+    master_sites    https://files.pythonhosted.org/packages/58/76/705b5c776f783d1ba7c630347463d4ae323282bbd859a8e9420c7ff79581/
+    distname        selenium-4.1.0-py3-none-any
+
+    checksums       rmd160  036d430dc7775f582e6606b27a20ff37cfa1c0c7 \
+                    sha256  27e7b64df961d609f3d57237caa0df123abbbe22d038f2ec9e332fb90ec1a939 \
+                    size    958787
+
+    extract.suffix  .whl
+    extract.only
+
+    use_configure   no
+
+    build           {}
+
+    destroot.cmd    pip-${python.branch}
+    destroot.args   --ignore-installed \
+                    --no-cache-dir \
+                    --no-dependencies \
+                    --root ${destroot}
+    destroot.post_args
+
+    destroot {
+        system "${destroot.cmd} ${destroot.target} ${destroot.args} ${distpath}/${distname}${extract.suffix}"
+    }
```

Comment:
are you sure this explicit `destroot` phase is needed. I would think that setting/overriding the appropriate values above would be sufficient to use just use the `destroot` phase that is set by `python` PG? Looking at, for example, the `py-mkl` port that seems to be the case...

Reply:
Thanks, corrected and done.